runbotics/runbotics-orchestrator-ui/src/src-app/components/Tile/ProcessTile/ProcessTile.utils.ts

```diff
@@ -1,5 +1,14 @@
-import { IProcess } from 'runbotics-common';
+import { IProcess, IProcessInstance, ProcessInstanceStatus } from 'runbotics-common';
 
 import { ProcessTab } from '#src-app/utils/process-tab';
 
 export const buildProcessUrl = (process: IProcess, tabName = ProcessTab.RUN) => `/app/processes/${process.id}/${tabName}`;
+
+export const checkActiveProcess = (processInstance: IProcessInstance) => {
```

Comment:
i suggest checkIsProcessActive -> suggest that the return value is boolean

Reply:
I'm no longer using that function.